lib/computer_player.rb

```diff
@@ -0,0 +1,16 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  def get_input
```

Comment:
'get_input' is a bit generic.  Can you think of a better way to describe this?  'throw' maybe?